presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -124,6 +124,8 @@
     private boolean writesToNonManagedTablesEnabled;
```

Comment:
Please make your commit message to follow: https://chris.beams.io/posts/git-commit/

Reply:
thanks for your reminder